packages/ComponentLibrary/src/components/Modal/index.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Modal from '@mui/material/Modal';
+import modalStyles from './modal.styles';
+
+interface ModalCustomProps {
+  height?: number;
+  width?: number;
+  posX?: number;
+  posY?: number;
+  children: React.ReactNode;
+}
+
+const ModalCustom: React.FC<ModalCustomProps> = ({
+  height,
+  width,
+  posX = 'center',
+  posY = 'center',
+  children,
+}) => {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  return (
+    <div>
+      <Button onClick={handleOpen}>Open modal</Button>
+      <Modal
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}>
+        <Box
+          sx={{
```

Comment:
Same here, take the styles to their corresponding file. Validate if it is correct that this component is centered in the center of the screen